### ISSUE TITLE
Templating: 'undefined' variable when using raw filter

### ISSUE THREAD

**Issue by Eraac** [grafana/grafana#26816]
**What happened**:
When I use a row with `repeat for` and use the variable in a panel inside the row like this `${myVar:raw}`, I get "undefined"

**What you expected to happen**:
The value of the variable instead of undefined

**How to reproduce it (as minimally and precisely as possible)**:
Use this dashboard
```
{
  "annotations": {
    "list": [
      {
        "builtIn": 1,
        "datasource": "-- Grafana --",
        "enable": true,
        "hide": true,
        "iconColor": "rgba(0, 211, 255, 1)",
        "name": "Annotations & Alerts",
        "type": "dashboard"
      }
    ]
  },
  "editable": true,
  "gnetId": null,
  "graphTooltip": 0,
  "id": null,
  "iteration": 1596647475324,
  "links": [],
  "panels": [
    {
      "datasource": null,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 0
      },
      "id": 2,
      "panels": [],
      "repeat": "list",
      "scopedVars": {
        "list": {
          "selected": true,
          "text": "1",
          "value": "1"
        }
      },
      "title": "$list",
      "type": "row"
    },
    {
      "datasource": null,
      "fieldConfig": {
        "defaults": {
          "custom": {}
        },
        "overrides": []
      },
      "gridPos": {
        "h": 8,
        "w": 12,
        "x": 0,
        "y": 1
      },
      "id": 4,
      "options": {
        "content": "${list:raw}",
        "mode": "markdown"
      },
      "pluginVersion": "7.1.0",
      "scopedVars": {
        "list": {
          "selected": true,
          "text": "1",
          "value": "1"
        }
      },
      "timeFrom": null,
      "timeShift": null,
      "title": "Panel Title",
      "type": "text"
    },
    {
      "datasource": null,
      "gridPos": {
        "h": 1,
        "w": 24,
        "x": 0,
        "y": 9
      },
      "id": 5,
      "panels": [],
      "repeat": null,
      "repeatIteration": 1596647475324,
      "repeatPanelId": 2,
      "scopedVars": {
        "list": {
          "selected": true,
          "text": "3",
          "value": "3"
        }
      },
      "title": "$list",
      "type": "row"
    },
    {
      "datasource": null,
      "fieldConfig": {
        "defaults": {
          "custom": {}
        },
        "overrides": []
      },
      "gridPos": {
        "h": 8,
        "w": 12,
        "x": 0,
        "y": 10
      },
      "id": 6,
      "options": {
        "content": "${list:raw}",
        "mode": "markdown"
      },
      "pluginVersion": "7.1.0",
      "repeatIteration": 1596647475324,
      "repeatPanelId": 4,
      "repeatedByRow": true,
      "scopedVars": {
        "list": {
          "selected": true,
          "text": "3",
          "value": "3"
        }
      },
      "timeFrom": null,
      "timeShift": null,
      "title": "Panel Title",
      "type": "text"
    }
  ],
  "schemaVersion": 26,
  "style": "dark",
  "tags": [],
  "templating": {
    "list": [
      {
        "allValue": null,
        "current": {
          "selected": true,
          "tags": [],
          "text": "1 + 3",
          "value": [
            "1",
            "3"
          ]
        },
        "hide": 0,
        "includeAll": false,
        "label": null,
        "multi": true,
        "name": "list",
        "options": [
          {
            "selected": true,
            "text": "1",
            "value": "1"
          },
          {
            "selected": false,
            "text": "2",
            "value": "2"
          },
          {
            "selected": true,
            "text": "3",
            "value": "3"
          }
        ],
        "query": "1,2,3",
        "queryValue": "",
        "skipUrlSync": false,
        "type": "custom"
      }
    ]
  },
  "time": {
    "from": "now-6h",
    "to": "now"
  },
  "timepicker": {
    "refresh_intervals": [
      "5s",
      "10s",
      "30s",
      "1m",
      "5m",
      "15m",
      "30m",
      "1h",
      "2h",
      "1d"
    ]
  },
  "timezone": "",
  "title": "New dashboard",
  "uid": null,
  "version": 0
}
```

**Anything else we need to know?**:
I've rollback to 7.1.1, no issue

**Environment**:
- Grafana version: 7.1.2
- OS Grafana is installed on: Kubernetes + official helm chart
- User OS & Browser: Chrome / MacOS

**Comment by svenstaro** [comment]
This can apparently be closed. I tested the fix #26818 and it appears to work just fine.

**Comment by zoltanbedi** [comment]
You are right @svenstaro, thanks for the heads up. This is fixed and released in 7.1.3 🎉
